lms/djangoapps/courseware/views.py

```diff
@@ -102,6 +102,7 @@
 template_imports = {'urllib': urllib}
 
 CONTENT_DEPTH = 2
+REQUIREMENTS_DISPLAY_MODES = CourseMode.CREDIT_MODES + [CourseMode.VERIFIED]
```

Comment:
@aamir-khan 
Please help me understand this. 

PR heading reads as `Updated the progress page to not display the eligibility in honor mode`, should we just skip credit requirements for `honor`? or we have clear instructions to ONLY display credit requirements for `credit` and `verified`?